server/sonar-server/src/main/java/org/sonar/server/organization/ws/RemoveMemberAction.java

```diff
@@ -96,6 +99,8 @@ public void handle(Request request, Response response) throws Exception {
 
       dbClient.organizationMemberDao().delete(dbSession, organizationMember.getOrganizationUuid(), organizationMember.getUserId());
       dbSession.commit();
+
+      userIndexer.index(user.getLogin());
```

Comment:
we want to redindex user permissions, don't we? I'm thinking about authorizations.